lib/transaction.js

```diff
@@ -0,0 +1,37 @@
+const apiRequest = require('./apiRequest');
+
+async function getTransactionListByUser() { 
+  const apiResponse = await apiRequest("GET", "/transaction/find-by-user", "?p=1", null)
+  return apiResponse;
+}
+
+async function createTransaction(receiver, amount, cause, meta_data) { 
+  // {\n\t\"receiver\": \"************\",\n\t\"amount\": 5000,\n\t\"cause\": \"\",\n\t\"meta_data\": []\n}
```

Comment:
use block comment if you want to indicate the response format in an example, or remove it at all. There will be enough API documentation to explain the request and response formats

Reply:
Will remove them